blurring_as_a_service/inference_pipeline/components/detect_and_blur_sensitive_data.py

```diff
@@ -115,23 +116,15 @@ def detect_and_blur_sensitive_data(
                     yolo_yaml_path, batch_file_txt
                 )  # use outputs folder as Azure expects outputs there
 
-                # Define the locked file path with ".lock" extension
-                locked_file_path = file_path + ".lock"
-
                 read_success = False
                 try:
```

Comment:
do we need this try except block here since the object has the try except also in enter and exit functions?

Reply:
Yes, this is due to how the code was designed and the `read_success = True` flag that we need to setup. I also don't like it at all I have to say.
Actually maybe it's not needed, I don't understand fully why I kept it now... mmm..

Reply:
If it fails, will it go out from the with or continue with the next line.
I'm afraid in some cases will continue and set the read success True when it should be false.